components/NavigationBar/src/MDCNavigationBar.h

```diff
@@ -99,18 +99,11 @@ IB_DESIGNABLE
 @property(nonatomic, strong, nullable) UIView *titleView;
 
 /**
- Display attributes for the titleView's title text.
-
- Setting this property will render an NSAttributedString with the assigned attributes across the
- entire text.
+ The font applied to the title of navigation bar.
+ Font size is enforced to 20.
+ Default is MDCTypography's titleFont.
  */
-#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
-@property(nonatomic, copy, nullable)
-    NSDictionary<NSAttributedStringKey, id> *titleTextAttributes UI_APPEARANCE_SELECTOR;
-#else
-@property(nonatomic, copy, nullable)
-    NSDictionary<NSString *, id> *titleTextAttributes UI_APPEARANCE_SELECTOR;
-#endif
+@property (nonatomic, strong, null_resettable) UIFont *titleFont;
```

Comment:
Also document how this property relates to `titleTextAttributes`. Notably, call out:

- `titleTextAttributes` will be deprecated in a future release and you should prefer `titleFont` instead.
- If a client sets both properties, document what the expected behavior is.